### DISCUSSION TITLE
Mirror mode: Mixpanel seems to keep staging tables in users' Redshift database

### DISCUSSION THREAD

**Discussion by v0rt3xh** [mixpanel/docs-legacy#1694] [General]
Our team started to explore [mirror mode](https://docs.mixpanel.com/docs/tracking-methods/warehouse-connectors#mirror) this November. To be more specific, we are testing this new feature on our AWS Redshift database. 

A recent incident might reveal a potential flaw of Mirror mode on Redshift destination. The number of tables in our database kept increasing and reached AWS' limit yesterday, which impacted our data pipeline and downstream tasks. 

We are aware of the fact that this Mirror mode is a relatively new feature and might still be under a beta testing framework. I haven't received responses from support@mixpanel.com nor has my colleague got advice from https://community.mixpanel.com/. This issue might cause a huge problem for other customers as well. Thus, I chose to post here. 

Feel free to route me to the correct place or the right support team. I also noticed that @myronkaifung added a callout [on this file](https://github.com/mixpanel/docs/commit/66ef878fab92b4eb08928da6e9f6a2d1b38bc9ae). So probably your team has started to work on resolving this issue. Thanks.

**FYI Attachment**
This image shows the increasing amount of tables in our Redshift database. *Periodic decrease: Amazon's scheduled maintenance. Most recent sharp drop: I manually deleted Mixpanel's staging tables*
<img width="1639" alt="image" src="https://github.com/user-attachments/assets/0d090740-4373-4148-afe2-e9fdb567d216" />

**Comment by myronkaifung** [comment]
Hi @v0rt3xh , thanks for raising the issue. My teammate Mikey just picked up the support ticket to work with you more directly on the issue. I linked this discussion to him so he has some additional context.
